thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js

```diff
@@ -31,14 +31,14 @@ export default Component.extend({
   //
   // external (optional)
   //
-  analysisRangeMax: moment().endOf('day'),
+  analysisRangeMax: moment().startOf('day').add(1, 'day'),
 
   analysisRangePredefined: {
-    'Today': [moment(), moment().endOf('day')],
-    'Last 3 days': [moment().subtract(2, 'days').startOf('day'), moment().endOf('day')],
-    'Last 7 days': [moment().subtract(6, 'days').startOf('day'), moment().endOf('day')],
-    'Last 14 days': [moment().subtract(13, 'days').startOf('day'), moment().endOf('day')],
-    'Last 28 days': [moment().subtract(27, 'days').startOf('day'), moment().endOf('day')]
+    'Today': [moment(), moment().startOf('day').add(1, 'hour')],
```

Comment:
nit: technically it's 'days' and 'hours', but moment handles this as well (not too sure if this is documented though) 

```
moment().add(1, 'days').calendar();  
// from their doc https://momentjs.com/
```